reports/summary-2020.md

```diff
@@ -0,0 +1,814 @@
+
+The following themes emerged from multiple community groups reporting in with 
+similar experiences - whether positive or challenging - and areas of research to
+explore more in the future. 
+
+### Project communication strategy 
+  
+During major growth phases, the Steering Committee needed the various [community
+groups to report in and out] regularly with the group's members, the
+project at large, and at KubeCons. This added up to many duplicate meetings and 
+many update slides. The
+COVID-19 pandemic and an increase in contributors outside of the
+North America Pacific Timezone made regular meetings difficult. Chairs and other
+project leads asked for a more streamline and consistent
+reporting and feedback mechanism. 
+    
+#### What we've done
+
+- Changed [Community Meeting] cadence from weekly to monthly, and changed 
+meeting style from a "read out" of updates to more discussion oriented.
+- Encouraged groups to use asychronous methods for delivering updates and 
+gathering feedback. For example: Slack "standup" theads which feed into larger 
+scale reporting to a group on a mailing list.
+- Created an internal marketing group under SIG Contributor Experience to help
+  facilitate community communication.
+
+
+[Community Meeting]: https://k8s.dev/events/community-meeting/
+[community groups to report in and out]: https://git.k8s.io/community/committee-steering/governance/sig-governance.md
+[Governance requirements]: https://git.k8s.io/community/committee-steering/governance/sig-governance.md#chair
+    
+#### Areas of research 
+
+- Review SIG charters to re-focus goals of each group at the current state of
+  maturity
+- Expand on async meeting adoption and establish best practices 
+- Improving subproject communication and connective glue back to the sponsoring
+  SIG
+- Clarify conditions for archiving or putting a subproject in maintenance mode
+- Connecting with the end user community and understanding what they want to
+  hear from upstream
+  
+    
+### Refining the Contributor Lifecycle
+
+There are a number of ways to define membership to the Kubernetes project
+and its community groups. The project's community groups must define
+their own membership terms and then various other activities and roles.
+Membership can drive:
+
+- Kubernetes GitHub Organization membership 
+- Consensus 
+- Voting
+
+**Project Membership**
+
+The Kubernetes project defines [membership] as contributors added to to one of 
+the four
+GitHub organizations. Membership grants a contributor access to test their pull
+requests, among other benefits. Membership also defines
+a member's role within the project based on their place on the [Contributor
+Ladder][ldr]. Membership does not grant voting rights for Steering Committe and
+governance matters; there is a separate set of criteria for [elections].
+
+There are two areas to address: early membership (onboarding) and sustainable 
+membership (ongoing contributor activity).
+
+**Early Membership**
```

Comment:
What is the point of this section? See my previous comment w.r.t. "Refining the contributor lifecycle" – What is the takeaway for the reader? Why do they need to read this section?